lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+
+
+  def initialize
```

Comment:
Consider open/closed principle. Maybe think of adding the list of food as an argument to the menu class so it is interchangeable - https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#open-closed-principle